### PR TITLE
Expander updates

### PR DESCRIPTION
**Pull request by SethDavenport**
* underline expander link on hover
* don't float the toggle link - now you can select a percentage width. Makes it work better in grids.
* use onInit for logic.

### CODE REVIEW

File: src/layout/expander/expander.controller.js [hunk 1]
```diff
@@ -1,8 +1,11 @@
 export default class ExpanderController {
-  constructor() {
+
+  $onInit = () => {
     this.showText = this.showText || '+';
     this.hideText = this.hideText || '-';
     this.expanded = this.expanded || false;
+    this.toggleWidth = `${this.toggleWidthPercent || '20'}%`;
+    this.headerContentWidth = `${100 - (this.toggleWidthPercent || 20)}%`;
```
Comment (benjyblack) [inline src/layout/expander/expander.controller.js]: I'd probably extract 20 into a constant so it can be changed in just one place

**Comment by benjyblack** [comment]
Would it be helpful to add an example of this option to the index page?

File: src/layout/expander/expander.controller.js (hunk 1 above)
Comment (SethDavenport) [inline src/layout/expander/expander.controller.js]: Sure will do.

**Comment by SethDavenport** [comment]
Yeah sure.  That page is getting a bit out of control but I suppose one more thing won't hurt :)

I'll split it up soon.

**Pull request merged**